python/rmm/__init__.py

```diff
@@ -28,4 +28,21 @@
     unregister_reinitialize_hook,
 )
 
+__all__ = [
+    "DeviceBuffer",
+    "RMMError",
+    "RMMNumbaManager",
+    "_numba_memory_manager",
```

Comment:
Should the name `rmm._numba_memory_manager` be in the public API? It was explicitly imported but appears unused. The source defines it and mentioned a link (below), but I couldn't find anything in Numba that was referencing `rmm._numba_memory_manager`. cc: @gmarkall this was last touched 2 years ago in https://github.com/rapidsai/rmm/pull/317.

https://github.com/rapidsai/rmm/blob/45f60da0fc6f95b226404fddf9f6d47ecf499f21/python/rmm/rmm.py#L205-L208

Reply:
It is needed to set Numba's memory manager to RMM with an environment variable, as noted in lines 205-207 above - this is documented in the EMM Plugin docs: https://numba.readthedocs.io/en/latest/cuda/external-memory.html#environment-variable.

One use case I use this for is to run the Numba test suite with RMM for allocation instead of Numba's internal memory management, like:

```
$ NUMBA_CUDA_MEMORY_MANAGER=rmm python -m numba.runtests numba.cuda.tests
```

(I'm open to changing the way we do this if you feel this is a bad design - it seemed reasonable at the time)

Reply:
Thanks, I was having difficulty finding that in the Numba docs! I think this is fine but it is a bit tricky to know what APIs are public. Is this environment variable supported for anything besides RMM? I would have preferred `NUMBA_CUDA_MEMORY_MANAGER=rmm.RMMNumbaManager` and importing based on that name.

Reply:
I'll leave the `import` but remove this name from `__all__` since it isn't meant to be a public API.
```suggestion
```

Reply:
> Is this environment variable supported for anything besides RMM?

It's supported for any EMM plugin - I'm not sure there are too many about, but I also wrote one for Numba to be able to use CuPy's allocator.

> I would have preferred `NUMBA_CUDA_MEMORY_MANAGER=rmm.RMMNumbaManager` and importing based on that name.

I think I had something like that in an early design and changed to the current incarnation - I can't quite recall why I made this change though.